src/app/models/table-models/data-table-state.ts

```diff
@@ -0,0 +1,14 @@
+import {Order} from "./order";
+
+export class DataTableState {
+  public rowDimensions: Array<string>;
```

Comment:
This would be an array of Dimensions, Array<Dimension>

Reply:
but in back-end we store only names of dimensions, we don't need additional info, that's why saving and restoring functions are using it as an array of strings

Reply:
backend design does not influence how we design table state, the same logic as the design for the query class.